lib/email_manager.js

```diff
@@ -61,6 +91,7 @@ exports.start = function (config_manager) {
       return transport.sendMail(email, function (err) {
         if (err) {
           // nodemailer errors are missing .message property
+          console.log(err)
```

Comment:
As for why no printing to stderr here: this is rather far down in the stack and we just report errors back up the call chain, so some code on a higher level can decide how and what to report to an end-user.